src/flowMC/sampler/Sampler.py

```diff
@@ -98,19 +69,10 @@ def __init__(
         self.optim_state = tx.init(eqx.filter(self.nf_model, eqx.is_array))
         self.nf_training_loop, train_epoch, train_step = make_training_loop(tx)
 
-        # Initialized result dictionary
```

Comment:
Since this code is pretty small anyway, I don't see the point of adding a new file and a new function for this. Revert this part.

Reply:
The idea was to reduce code duplication since it is used for both the training and production summaries. I have moved the function to the Sampler file, please let me know if you prefer to revert that as well.

Reply:
Since the code is quite short here and it is only used twice (and intended to be only be used twice), I think having it being explicit like it is currently is preferred. Also, in general I would avoid string as a input for bookkeeping function, they can lead to all kinds of instability

Reply:
Ok, thanks for the feedback and insights into the code design! Reverted this change.